src/util/FactorUtil.js

```diff
@@ -86,6 +86,12 @@ define(['okta'], function (Okta) {
       description: Okta.loc('factor.windowsHello.description', 'login'),
       iconClassName: 'mfa-windows-hello',
       sortOrder: 10
+    },
+    'U2F': {
```

Comment:
Should probably check with @karlmcguinness-okta or @hassenkaraa-okta to see what the new sort order should be - for example, I don't think Windows Hello or U2F should be below Security Question.